modules/gltf/src/lib/types/gltf-ext-mesh-features-schema.ts

```diff
@@ -0,0 +1,53 @@
+import {GLTFTextureInfoMetadata} from './gltf-json-schema';
+/**
+ * Spec - https://github.com/CesiumGS/glTF/blob/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features/schema/featureId.schema.json
+ */
+
+export const EXTENSION_NAME_EXT_MESH_FEATURES = 'EXT_mesh_features';
+
+/**
+ * EXT_mesh_features extension types
+ * This is a primitive-level extension
+ * Spec - https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features
+ *
+ * JSON Schema - https://github.com/CesiumGS/glTF/blob/c38f7f37e894004353c15cd0481bc5b7381ce841/extensions/2.0/Vendor/EXT_mesh_features/schema/mesh.primitive.EXT_mesh_features.schema.json
+ * An object describing feature IDs for a mesh primitive.
+ */
+export type GLTF_EXT_mesh_features = {
+  /** An array of feature ID sets. */
+  featureIds: GLTF_EXT_mesh_features_featureId[];
+  extensions?: Record<string, any>;
+  extras?: any;
+};
+
+/**
+ * JSON Schema https://github.com/CesiumGS/glTF/blob/c38f7f37e894004353c15cd0481bc5b7381ce841/extensions/2.0/Vendor/EXT_mesh_features/schema/featureId.schema.json
+ * Feature IDs stored in an attribute or texture.
```

Comment:
Nit : put the link last
```suggestion
  * Feature IDs stored in an attribute or texture.   
  * @see https://github.com/CesiumGS/glTF/blob/c38f7f37e894004353c15cd0481bc5b7381ce841/extensions/2.0/Vendor/EXT_mesh_features/schema/featureId.schema.json

```